src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -24,9 +24,10 @@ namespace AppInstaller::Runtime
         constexpr std::string_view s_SecureSettings_Base = "Microsoft/WinGet"sv;
         constexpr std::string_view s_SecureSettings_UserRelative = "settings"sv;
         constexpr std::string_view s_SecureSettings_Relative_Unpackaged = "win"sv;
-        constexpr std::string_view s_PortableAppUserRoot = "Microsoft/WinGet"sv;
+        constexpr std::string_view s_PortableAppUserRoot = "Microsoft\\WinGet"sv;

```

Comment:
VC `filesystem` supports both, it is probably in the spec but I'm not sure.

Reply:
I was having some issues with modified path showing up with the wrong path separator like so: `AppData\Local\Microsoft/WinGet\Links;` 

I decided to just separate them out into their own strings for clarity.